admin/meetings/views.py

```diff
@@ -0,0 +1,129 @@
+        try:
+            self.conf = Conference.get_by_endpoint(endpoint, active=False)
+        except ConferenceError:
+            raise Http404('Meeting with endpoint "{}" not found'.format(
+                endpoint
+            ))
+        return super(MeetingFormView, self).dispatch(request, *args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        kwargs.setdefault('endpoint', self.kwargs.get('endpoint'))
+        return super(MeetingFormView, self).get_context_data(**kwargs)
+
+    def get_initial(self):
+        self.initial = serialize_meeting(self.conf)
+        self.initial.setdefault('edit', True)
+        return super(MeetingFormView, self).get_initial()
+
+    def form_valid(self, form):
+        custom_fields, data = get_custom_fields(form.cleaned_data)
+        if 'admins' in form.changed_data:
+            admin_users = get_admin_users(data.get('admins'))
+            self.conf.admins = admin_users
+        self.conf.name = data.get('name')
+        self.conf.info_url = data.get('info_url')
+        self.conf.logo_url = data.get('logo_url')
+        self.conf.active = data.get('active')
+        self.conf.public_projects = data.get('public_projects')
+        self.conf.poster = data.get('poster')
+        self.conf.talk = data.get('talk')
+        self.conf.field_names.update(custom_fields)
+        self.conf.save()
+        return super(MeetingFormView, self).form_valid(form)
+
+    @property
+    def success_url(self):
+        return reverse('meetings:detail',
+                       kwargs={'endpoint': self.kwargs.get('endpoint')})
+
+
+class MeetingCreateFormView(OSFAdmin, FormView):
+    template_name = 'meetings/create.html'
+    form_class = MeetingForm
+
+    def get_initial(self):
+        default_field_names = Conference.DEFAULT_FIELD_NAMES
+        self.initial.update({'field_{}'.format(k): default_field_names[k]
+                             for k in default_field_names.keys()})
+        self.initial.setdefault('edit', False)
+        return super(MeetingCreateFormView, self).get_initial()
+
+    def form_valid(self, form):
+        custom_fields, data = get_custom_fields(form.cleaned_data)
+        endpoint = data.pop('endpoint')
+        self.kwargs.setdefault('endpoint', endpoint)
+        # Form validation already checks emails for existence
+        admin_users = get_admin_users(data.pop('admins'))
+        # Form validation already catches if a conference endpoint exists
+        new_conf = Conference(
+            endpoint=endpoint,
+            admins=admin_users,
+            **data
+        )
+        new_conf.field_names.update(custom_fields)
+        new_conf.save()
+        return super(MeetingCreateFormView, self).form_valid(form)
+
+    def get_success_url(self):
+        return reverse('meetings:detail',
+                       kwargs={'endpoint': self.kwargs.get('endpoint')})
+
+
+def get_custom_fields(data):
+    """Return two dicts, one with 'field' stripped
```

Comment:
The fields are in a dictionary. I am basically bringing them to the top so that they can be form fields, but can be distinguished easily as going back into the dictionary. Another way would be to have a map between the names and their membership in the dictionary. Preference @sloria?